0xProject/protocol
# Feat/extended quote report

## Description
This PR adds an Extended Quote report to `asset-swapper`. This is always computed (instead of only on RFQ trades) to allow for better data analysis.

A "new" quote report was used to have both the regular and extended running during the transition time.

## Testing instructions
Calling `getOptimizerResultAsync` should return the additional (optional) field `extendedQuoteReport` 

See https://github.com/0xProject/0x-api/pull/724

## Types of changes





 * New feature (non-breaking change which adds functionality)



## Checklist:
